testssl.sh

```diff
@@ -1770,26 +1770,50 @@ asciihex_to_binary_file(){
      local -i len
      local -i i ip2 ip4 ip6 ip8 ip10 ip12 ip14
      local -i remainder
+     if [[ $file == /dev/stdout ]]; then
+          local stdout="1"
+     else
+          local stdout="0"
+     fi
+
```

Comment:
You're doing a check to populate a variable and later on you do again a check with this variable.

I think it would be easier to just you below a statement like ``if [[ $file == /dev/stdout ]]``